### DISCUSSION TITLE
Not able to disable google auth

### DISCUSSION THREAD

**Discussion by sweekark** [Budibase/budibase#9348] [Help] [answered]
![image](https://user-images.githubusercontent.com/3829871/212606849-6411dfcb-03f5-472f-8d14-f2cfe3fc38d6.png)
although the toogle is turned off for google auth 

its not getting saved.

**Comment by andz-bb** [comment, marked as answer]
hey @sweekark 

Thanks for bringing this to our attention, it's a validation issue. I've raised this as a bug here: #9351 

As a temporary workaround, if you enter any text into the Client ID and Client Secret fields you will be able to save the deactivation properly.
